docker/ubuntu_22.04-dev.dockerfile

```diff
@@ -169,45 +118,19 @@ ENV PYNE_DAGMC_ARGS "--dagmc $HOME/opt/dagmc"
 FROM dagmc AS openmc
 ARG build_hdf5
 # build/install OpenMC Python API
-RUN if [ "$build_hdf5" != "NO" ]; then \
-            export HDF5_ROOT="$HDF5_INSTALL_PATH" ; \
-    fi ;\
+RUN export HDF5_ROOT="$HDF5_INSTALL_PATH" ; \
     git clone https://github.com/openmc-dev/openmc.git $HOME/opt/openmc \
     && cd  $HOME/opt/openmc \
-    && git checkout tags/v0.13.0 \
+    && git checkout tags/v0.14.0 \
```

Comment:
We could make the HDF5 version an ARG which defaults to 0.14.0

Reply:
This is an openmc version, but yes, your point is well-taken